crates/api/src/instance.rs

```diff
@@ -170,24 +159,17 @@ impl Instance {
         self.module.store()
     }
 
-    /// Returns the associated [`Module`] that this `Instance` instantiated.
-    ///
-    /// The corresponding [`Module`] here is a static version of this `Instance`
-    /// which can be used to learn information such as naming information about
-    /// various functions.
-    pub fn module(&self) -> &Module {
-        &self.module
-    }
-
     /// Returns the list of exported items from this [`Instance`].
-    ///
-    /// Note that the exports here do not have names associated with them,
-    /// they're simply the values that are exported. To learn the value of each
-    /// export you'll need to consult [`Module::exports`]. The list returned
-    /// here maps 1:1 with the list that [`Module::exports`] returns, and
-    /// [`ExportType`](crate::ExportType) contains the name of each export.
-    pub fn exports(&self) -> &[Extern] {
-        &self.exports
+    pub fn exports<'me>(&'me self) -> impl ExactSizeIterator<Item = Export> + 'me {
```

Comment:
I think that this is auto-inferred, but could this be changed to `Export<'me>`?

Reply:
Done